File: dotnet/01-commands/fix/DemoFix/MyClassExample2.cs
namespace DemoFix
{
    internal class MyClassExample2
    {
        
        public int i;
        public void Method() { }
        public int Prop
        {
            get
            {
                return 1;
            }
        }

        public static void Main()
        {
            i = 10;
            Method();  
            int p = Prop;   
        }
    }
}

File: nodejs/01-commands/fix/sourcecode/MyClassExample2.js
class MyClassExample2 {
    constructor() {
        this.i = 0;
    }

    method() {
        // Method implementation
    }

    getProp() {
        return 1;
    }

    main() {
        this.i = 10;
        this.method();
        const p = this.getProp();
        console.log(`i: ${this.i}, p: ${p}`);
    }
}

const instance = new MyClassExample2();
instance.main();

module.exports = MyClassExample2;